packages/meditrak-server/src/routes/CRUDHandler/CRUDHandler.js

```diff
@@ -21,5 +22,6 @@ export class CRUDHandler extends RouteHandler {
     this.parentResource = params.parentResource;
     this.parentRecordType = resourceToRecordType(this.parentResource);
     this.parentRecordId = params.parentRecordId;
+    this.model = this.models[singularise(this.resource)];
```

Comment:
```suggestion
    this.resourceModel = this.models[singularise(this.resource)];
```
What do you reckon? Seeing it in the context of the fields above, this seems to be less ambiguous

Reply:
Good call, having `models` and `model` at the same level was probably asking for mixups.